docs/pages/packages/moments/FetchMoments.mdx

```diff
@@ -0,0 +1,57 @@
+# Fetch Moments
+
+## List Moments
+
+Fetching moments per page are done by giving an `offset` and a `limit`. This way
+all the uploaded moments could be retrieved.
+
+```typescript
+import { Moment } from '@poap-xyz/moments';
+import { Order, PaginatedResult } from '@poap-xyz/utils';
+
+const paginatedMoments: PaginatedResult<Moment> = await client.fetch({
+  offset: 0,
+  limit: 10,
+  idOrder: Order.DESC,
+});
+```
+
+### By Drop
+
+The list can be filtered by one or more Drops by giving `drop_ids` list with the
+Drop IDs. Also, the list can be sorted by 'id', 'created', 'tokenId' or `dropId`.
```

Comment:
`id`, `created`, `tokenId`

Reply:
Done in [a32760a](https://github.com/poap-xyz/poap.js/pull/130/commits/a32760ab667feeb7d1a0d1b1f6e039d53640e1d5).